universal-login-sdk/src/core/states/BlockState.ts

```diff
@@ -0,0 +1,18 @@
+import {BlockchainService} from '@universal-login/contracts';
+import {ReferenceCountedState} from './ReferenceCountedState';
+
+export class BlockState extends ReferenceCountedState<number> {
```

Comment:
`BlockNumberState` would probably be more descriptive

Reply:
I think it will be BlockNumberService in the end of the day, so i will rename it later.